### ISSUE TITLE
OAuth credentials are stored in the DB instead of the secrets vault

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#1673]
@revi's comment at <https://github.com/liberapay/liberapay.com/issues/943#issuecomment-588296218> made me realize that we should probably move our OAuth credentials out of the database and into the “secrets manager” (where we store our encryption keys).

**Comment by Changaco** [comment]
This could help avoid reports like <https://hackerone.com/reports/837733>, but we would have to rewrite the Git history. (cc @EdOverflow)

**Comment by EdOverflow** [comment]
@Changaco, we could use https://rtyley.github.io/bfg-repo-cleaner/ to remove the credentials from our Git history.

**Comment by Changaco** [comment]
That tool looks nice. However, rewriting the `master` branch can create various problems, so I'm not sure it's a good idea to do it only in the hope of avoiding a few HackerOne reports.

**Comment by EdOverflow** [comment]
Good point. Maybe what I will do instead is add triggers for those credentials on H1 so that reporters on HackerOne get an alert warning them those credentials are just for testing purposes.
